common/middleware/set-primary-navigation.test.js

```diff
@@ -199,6 +221,48 @@ describe('#setPrimaryNavigation()', function() {
               href: '/moves/outgoing',
               text: 'primary_navigation.outgoing',
             },
+            {
+              active: false,
+              href: '/moves/incoming',
+              text: 'primary_navigation.incoming',
+            },
+          ])
+        })
+      })
+
+      context('on incoming page', function() {
```

Comment:
This is becoming a big file. Maybe split it between the lines of outgoing and incoming?

Reply:
Not sure what you mean by splitting it, could you explain a little further or post an example?

It also felt a little repetitive to me testing the highlight tab, I wonder if I could look at checking the number of `active = false` items and then just check that the `active = true` item matches the one expected and if that would be enough. What do you think?

Reply:
Have pushed a fixup, let me know what you think

Reply:
```
set-primary-navigation.test.js 

=>

set-primary-navigation.test.js
set-primary-navigation.ougoing.test.js
set-primary-navigation.incoming.test.js
```
I'm assuming that's what you meant B? (And if so, I'm all in favour of that)

Reply:
Hmm, I'm not sure I like the idea of that because it breaks the pattern of having a test file per file. Making it easy to see when scanning directories when test files are missing.

That's the main reason I prefer the pattern of side-by-side test files over a `/test` directory. 

I think that pattern would make sense if the main file was also split but just splitting a test file feels a little odd.

Reply:
@solidgoldpig yes, that's what I meant :-)